tests/Agent/IntegrationTests/ContainerApplications/KafkaTestApp/Dockerfile

```diff
@@ -0,0 +1,57 @@
+ARG DISTRO_TAG
+FROM --platform=amd64 mcr.microsoft.com/dotnet/aspnet:7.0-bullseye-slim AS base
+WORKDIR /app
+EXPOSE 80
+RUN apt-get update \
```

Comment:
Not sure I understand why we need to do all this work - looks like you're installing ASPNETCore 6, when we're already running an image that ASPNETCore 7 on it?